### DISCUSSION TITLE
Mention `pydantic-numpy` in the docs?

### DISCUSSION THREAD

**Discussion by caniko** [pydantic/pydantic#4316] [Projects using Pydantic]
Solves: #3516 #667

I think [`pydantic-numpy`](https://github.com/cheind/pydantic-numpy) is useful to every project that utilizes Pydantic with NumPy arrays as fields, and, soon, computed fields. I think it should at least be mentioned in the Pydantic docs.

Alternatively, Pydantic could subsume `pydantic-numpy`.

Disclaimer, I am a contributor to `pydantic-numpy`.

**Comment by samuelcolvin** [comment]
Looks really interesting, great work.

I've changed the category here for a start.

I think we should add a section to the docs of packages using pydantic.

What should we add? Obviously FastAPI? I guess we can just work down https://github.com/topics/pydantic for a start.

PR welcome to start this and add the top 10 from that list.

**Reply by caniko** [reply]
Sounds good, I'll do it in the weekend :)
